2996_reflection/reflection.md

```diff
@@ -846,25 +846,25 @@ On Compiler Explorer: [EDG](https://godbolt.org/z/W74qxqnhf), [Clang](https://go
 #include <meta>
 #include <array>
 
-template <typename T, std::size_t N>
-struct struct_of_arrays_impl;
+template <typename T, size_t N>
+constexpr auto struct_of_arrays_impl = [] consteval {
```

Comment:
I don't think we should need this `consteval` right? Doesn't hurt, but shouldn't be strictly necessary?

Reply:
Sorry, the godbolt links have a better version. Replaced, done.